player/lua.c

```diff
@@ -1336,7 +1336,7 @@ static void add_functions(struct script_ctx *ctx)
 }
 
 const struct mp_scripting mp_scripting_lua = {
-    .name = "lua script",
+    .name = "lua",
```

Comment:
From what I can see, these were used only for cosmetic purposes and so this probably doesn't have any unintended side-effects. But maybe someone can double check it just to be sure.

Reply:
This actually would change the logging message to something, imo, a bit less nice. eg. it would be `Loading lua @osc.lua...` instead of `Loading lua script @osc.lua...`. Logically it makes more sense for this to be named "lua" though and for the debugging messages to be adjusted instead.